### PR TITLE
Added loginHint to login method

### PR DESCRIPTION
**Pull request by alex-minaiev-frontegg**

### CODE REVIEW

File: Sources/FronteggSwift/embedded/FronteggWebView.swift [hunk 1]
```diff
@@ -14,10 +14,11 @@ import AuthenticationServices
 public struct FronteggWebView: UIViewRepresentable {
     public typealias UIViewType = WKWebView
     private var fronteggAuth: FronteggAuth
+    private let loginHint: String?
```
Comment (frontegg-david) [inline Sources/FronteggSwift/embedded/FronteggWebView.swift:17]: move loginHint to fronteggAuth Service as we have for pendingLink option

File: Sources/FronteggSwift/embedded/EmbeddedLoginPage.swift [hunk 2]
```diff
@@ -11,16 +11,19 @@ import UIKit
...
+    }
     
     
     public var body: some View {
         ZStack {
             NavigationView{
                 VStack(alignment: .center) {
                     ZStack{
-                        FronteggWebView()
+                        FronteggWebView(loginHint: loginHint)
```
Comment (frontegg-david) [inline Sources/FronteggSwift/embedded/EmbeddedLoginPage.swift:26]: Do not add any constuctor parameter to frontegg views

**Review by frontegg-david — CHANGES_REQUESTED** [review]

File: Sources/FronteggSwift/embedded/FronteggWebView.swift (hunk 1 above)
Comment (alex-minaiev-frontegg) [reply Sources/FronteggSwift/embedded/FronteggWebView.swift]: Done

File: Sources/FronteggSwift/embedded/EmbeddedLoginPage.swift (hunk 2 above)
Comment (alex-minaiev-frontegg) [reply Sources/FronteggSwift/embedded/EmbeddedLoginPage.swift]: Done

**Review by frontegg-david — APPROVED** [review]

**Pull request merged**